features/FxA-56-email-deliverability/README.md

```diff
@@ -1,66 +1,160 @@
+fail to due email delivery issues.
+
+Future attempts to improve email deliverability
+can be judged successful
+if they improve this completion rate.
+
+### Lost Mail Rate for Email Flows
+
+For flows that result in an email being sent,
+we will produce a graph showing the percentage for which:
 
-There are different parts that will help us solve our email deliverability problems:
+* We receive a successful delivery notification from SES
+* We receive a bounce notification from SES
+* we receive no further information about delivery
 
-* Adding flow events for bounces
+The graph will look something like this:
 
-The plan is to add flow event metrics to our existing bounce tracking scripts in the `fxa-auth-server`. This approach will help us create a funnel metrics graph.
+![Mock graph for lost mail rate of email flows](lost_mail_rate.jpg)
 
-* Creating a "complaint rate" and a "bounce rate" dashboard
+This will help us to know whether deliverability issues
+are caused by issues downstream, upstream, or inside of SES.
 
-This dashboard would include some of the details we get
-from Amazon Web Services.
+### Bounce and complaint rate by template
 
-We can break down the dashboards by the email template and / or language where possible.
+We will produce a graph showing the number of bounce notifications per day,
+as a percentage of the number of emails sent that day,
+broken down by email template name.
+It will look something like this:
 
-Additionally, we can keep a record in our database, creating a table recording bounced emails. We can evolve what we do with records in the database as we learn.
+![Mock graph for bounce mail by template](bounce_rate_by_template.jpg)
 
-A simple first step we can do is when a user enters an email address that has a recent record in our bounce table, we can notify the user that the email had bounced, maybe check the spelling and/or that their email account isn't full, etc.
+We will produce a similar graph for complaint notifications.
 
-* Removing bad emails
+This will help us to determine if any particular type of email
+is more likely to be rejected, which may indicate some sort of
+email-content-related issue.
 
-If the email is not valid and keeps bouncing then we remove it from our system or at least stop sending emails to it.
+### Bounce and complaint rate by domain
 
-* Check domain MX Records
+We will produce a graph showing the number of bounce notifications per day,
+as a percentage of the number of emails sent that day,
+broken down by email domain.
+It will look something like this:
+
+![Mock graph for bounce mail by domain](bounce_rate_by_domain.jpg)
+
+We will produce a similar graph for complaint notifications.
+
+This will help us to determine if we're having deliverability issues
+with a specific provider, which will aid in debugging.
+
+
+
+----
+
+## Detailed design
 
-We need to add a way to check against bad MX records when users sign up for new accounts in FxA.
```

Comment:
I think we should keep the mailcheck stuff in its own feature card, as a follow-up tactic once we've got these metrics in place.  I'm open to being convinced otherwise though.